runners/direct-java/src/test/java/org/apache/beam/runners/direct/CloningBundleFactoryTest.java

```diff
@@ -28,18 +28,21 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Iterables;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
 import org.apache.beam.runners.direct.DirectRunner.CommittedBundle;
 import org.apache.beam.runners.direct.DirectRunner.UncommittedBundle;
-import org.apache.beam.runners.direct.EncodabilityEnforcementFactoryTest.Record;
-import org.apache.beam.runners.direct.EncodabilityEnforcementFactoryTest.RecordNoDecodeCoder;
-import org.apache.beam.runners.direct.EncodabilityEnforcementFactoryTest.RecordNoEncodeCoder;
+import org.apache.beam.sdk.coders.AtomicCoder;
```

Comment:
+1 to this change and also in the future don't import between test suites, but break out the utility classes.